### PR TITLE
Thursday cullen

### PR DESCRIPTION
**Pull request by Skiwilliams**
is good changes

### CODE REVIEW

File: Scheduler.h [hunk 1]
```diff
@@ -49,4 +54,9 @@ void decrementTimer();
 */
 void releaseProcs();
 
+private:
+  std::priority_queue<Job> procaQueue;
```
Comment (GarrickHutheson) [inline Scheduler.h]: "procaQueue because it sounds kind of like pikachu" - cullen 2018

File: Scheduler.h [hunk 2]
```diff
@@ -56,9 +61,9 @@ void decrementTimer();
 */
 void releaseProcs();
 
-private:
+//private:
   std::priority_queue<int,Job,std::greater<int> > procaQueue;
-
+  int theFinalCountdown;
```
Comment (Skiwilliams) [inline Scheduler.h]: it's a timer

Comment (GarrickHutheson) [inline Scheduler.h]: "good code should document it self" - some pretentious asshole

File: Scheduler.h [hunk 3]
```diff
@@ -49,4 +61,10 @@ void decrementTimer();
 */
 void releaseProcs();
 
+//private:
+  std::priority_queue<int,Job,std::greater<int> > procaQueue;
+  int theFinalCountdown;
```
Comment (GarrickHutheson) [inline Scheduler.h]: no

**Pull request merged**